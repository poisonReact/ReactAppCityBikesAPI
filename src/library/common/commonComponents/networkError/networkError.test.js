import React from 'react'
import renderer from 'react-test-renderer'
import NetworkError from 'library/common/commonComponents/networkError/networkError'

it('render correctly NetworkError component', () => {
    const NetworkErrorComponent = renderer.create(<NetworkError />).toJSON()
    expect(NetworkErrorComponent).toMatchSnapshot()
})