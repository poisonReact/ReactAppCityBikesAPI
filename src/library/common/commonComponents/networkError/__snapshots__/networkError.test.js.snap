// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly NetworkError component 1`] = `
<div
  className="networkErrorWrapper"
>
  Some network error, please try again later
</div>
`;
